src/impl/zoneUtil.js

```diff
@@ -17,7 +18,8 @@ export function normalizeZone(input, defaultZone) {
     return input;
   } else if (isString(input)) {
     const lowered = input.toLowerCase();
-    if (lowered === "local" || lowered === "system") return defaultZone;
+    if (lowered === "local" || lowered === "default") return defaultZone;
```

Comment:
"local" is a legacy name for "system", so local should have followed system over to the new condition

Reply:
@icambron This has been refactored, local condition moved to the system one.